### DISCUSSION TITLE
How to cite this package

### DISCUSSION THREAD

**Discussion by AnasAlzahrani** [moj-analytical-services/splink#192] [General]
I'm planing to to publish an academic article that I used Splink package. How you would like to cite this package?
Thanks

**Comment by RobinL** [comment]
I'm not sure about the citation style but I'd go with:
Year: 2021 (for the latest version, 2020 for the earlier versions)
Title: Splink
Authors:  Robin Linacre , Sam Lindsay, Theodore Manassis
Organisation:  Ministry of Justice (United Kingdom)
If you can include a URL, please use https://github.com/moj-analytical-services/splink

I'd love to know more about your work! - if you have a moment feel free to drop me a line at robin.linacre [at] digital.justice.gov.uk

**Reply by RobinL** [reply]
@AnasAlzahrani did this even get published?  We'd love to hear about any published papers that used Splink (irrespective of whether there's a specific citation, we just like hearing about where work has benefitted from Splink)

**Reply by AnasAlzahrani** [reply]
We are still working on publishing. I'll definitely update you when we publish. Thanks a lot for your efforts.

**Reply by RobinL** [reply]
> We are still working on publishing. I'll definitely update you when we publish. Thanks a lot for your efforts.

Ok, great - thanks for the update!

**Reply by christiansegercrantz** [reply]
Hey! I'm currently writing my master thesis on record linkage and Splink is playing a essential role in it. Just a heads up, deadline is end of the month so should be out in a few months! @RobinL

**Reply by RobinL** [reply]
Amazing!  Looking forward to seeing it!!  Also feel free to give me a shout if you have any questions
